minkyu/app.js

```diff
@@ -31,24 +30,57 @@ app.use(cors());
 app.use(morgan('tiny'));
 
 app.get("/ping", (req,res) =>{
-    res.status(200).json({"message" : "pong"});
+    res.json({message : "pong"});
 });
-        // 유저 회원가입
-app.post("/users", async(req, res) =>{
-    const { name, email, password } = req.body;
-
+    // 게시글 등록하기
+app.post("/posting", async(req, res) => {
+    const {title, content, userId} = req.body;
+    console.log(title, content, userId);
     await database.query(
-        `INSERT INTO users(
-            name,
-            email,
-            password
-        ) VALUES (?, ?, ?);
+        `INSERT INTO posts(
+            title,
+            content,
+            user_id
+        ) VALUES (?, ?, ?)
         `,
-        [ name, email, password ]
+        [ title, content, userId ]
     );
-    res.status(200).json({ message : "successfully created" });
+    res.status(200).json({ message : "postCreated" });
```

Comment:
```suggestion
    res.status(201).json({ message : "postCreated" });
```
수정, 등록이라면 201 status code가 더 적합할 것 같네요!